dropshot/tests/test_demo.rs

```diff
@@ -774,6 +814,25 @@ async fn demo_handler_delete(
     Ok(HttpResponseDeleted())
 }
 
+#[endpoint {
+    method = GET,
+    path = "/testing/headers",
+}]
+async fn demo_handler_headers(
+    _rqctx: RequestCtx,
+) -> Result<HttpResponseHeaders<HttpResponseUpdatedNoContent>, HttpError> {
```

Comment:
This is cool.  And if you have a body to send back, I imagine this is like `HttpResponseHeaders<HttpResponseOk<SomeType>>`?

Reply:
Exactly right